app/views/hub.nunjucks

```diff
@@ -105,28 +80,13 @@
   <!--   <li>Stop smoking clinic</li> -->
   <!--   <li>Travel injections</li> -->
   <!-- </ul> -->
-  <hr class="hr--no-color"/>
+  <hr class="hr--fullwidth"/>
 
-  <h2>Parking and accessibility</h2>
-  <p>coming soon</p>
-  <!-- <ul class="link-list"> -->
-  <!--   <li>8 parking spaces, including 2 disabled parking spaces</li> -->
-  <!--   <li>Wheelchair access</li> -->
-  <!--   <li>Disabled toilet</li> -->
-  <!--   <li>Step-free access</li> -->
-  <!-- </ul> -->
-  <hr class="hr--no-color"/>
+  {% include "includes/parking-accessibility.nunjucks" %}
+  <hr class="hr--fullwidth"/>
```

Comment:
This line needs to be moved inside `includes/parking-accessibility.nunjucks` to prevent double rules like this profile http://profiles-pr-75.dev.c2s.nhschoices.net/gp-surgeries/107891